### ISSUE TITLE
svutRun requires git to be installed

### ISSUE THREAD

**Issue by sevetseh28** [dpretet/svut#28]
Hi, 

I wanted to report this issue im having. It seems `svutRun` is calling `git` cmd behind the scenes which is not installed in my environment. 

I think it should not be required.

```python
Traceback (most recent call last):
  File "/home/vscode/.local/bin/svutRun", line 448, in <module>
    main()
  File "/home/vscode/.local/bin/svutRun", line 386, in main
    git_tag = get_git_tag()
              ^^^^^^^^^^^^^
  File "/home/vscode/.local/bin/svutRun", line 315, in get_git_tag
    git_tag = subprocess.check_output(["git", "describe", "--tags", "--abbrev=0"])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.12/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
```

**Comment by dpretet** [comment]
Hello, 

Thanks to report the issue, this is definitely something I can fix. Can I ask you how did you install SVUT and on which platform ?

Damien

**Comment by dpretet** [comment]
Thank you for your feedback, it will be fixed in v1.9.2 :)
